CHANGELOG.md

```diff
@@ -1,42 +1,22 @@
 # Changelog
 
-All notable changes to Docker and Docker Compose resources for WSO2 Identity and Management version `6.1.x`
+All notable changes to Docker and Docker Compose resources for WSO2 Identity and Management version `7.0.x`
 in each resource release, will be documented in this file.
 
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 
-## v6.1.0.5 - 2024-02-28
-
-### Changed
-- JDK17 support.
-
-## v6.1.0.4 - 2024-02-12
-
-### Changed
-- Update alpine version 3.16.0 to 3.16.
-
-## v6.1.0.3 - 2023-10-19
-
-### Changed
-- Update jdk-11.0.19+7 to jdk-11.0.20.1+1.
-
-## v6.1.0.2 - 2023-06-27
-
-### Changed
-- Update jdk-11.0.18+10 to jdk-11.0.19+7.
-
-## v6.1.0.1 - 2023-02-01
+## v7.0.0.1 - 2024-03-01
```

Comment:
Update the date

Reply:
Addressed with https://github.com/wso2/docker-is/pull/395/commits/8f904a9a31b4c126917f0a7c86f9ef7c4e94fac7